### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by subbu1993**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # sd-serverdiagnostics
 Server Connector For MageDiagnostics
+
+Installation Process:
```
Comment (mpchadwick) [inline README.md]: Make this a header

```
## Installation
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+- Copy the repo contents to the /opt folder on each webserver
```
Comment (mpchadwick) [inline README.md]: Rather than "copy" I'd specifically say...

> `git clone` the repo in the `/opt` folder on each webserver.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+- cp .env.sample .env
+
+- In the .env file, set the Server key from the server key value found for that client in the magediagnostics dashboard
+
+- In the .env file, set the `SD_MAGEDIAGNOSTICS_API_ENDPOINT` to `https://magediagnostics.com:1234/api/v1/server`
+
+- Add a cron tab entry `0 0 * * * sh main.sh`
```
Comment (mpchadwick) [inline README.md]: should be...

```
0 0 * * * /opt/sd-serverdiagnostics/main.sh
```

Need the correct path. You don't need `sh` (the shebang specifies how it should be executed).

**Comment by subbu1993** [comment]
@mpchadwick all done!!!

**Pull request merged**